src/main/java/patio/voting/services/internal/VotingSchedulingService.java

```diff
@@ -98,7 +103,13 @@ public void scheduleVoting() {
   @Transactional
   /* default */ void checkVoting() {
     LOG.info("checking voting creation");
-    this.findAllToCreateVotingFrom().map(this::createVoting).forEach(this::notifyMembers);
+    this.findAllToCreateVotingFrom()
+        .map(
+            (group) -> {
```

Comment:
Why changing from:

```java
.map(this::createVoting)
```
to
```java
.map(
    (group) -> {
         var voting = this.createVoting(group);
         return voting;
      }
)
```

Reply:
Probably a quick test I did that I miss to restore it after the try. I'll bring it back to the original code